client-v2/src/shared/components/input/InputCheckboxToggle.tsx

```diff
@@ -66,16 +66,18 @@ const Checkbox = styled('input')`
 type Props = {
   label?: string;
   id: string;
+  dataTestId: string;
```

Comment:
In order to push the `dataTestId` down to the right place I needed to put this in the props and send it down to the `InputContainer`.